app/models/manifestation.rb

```diff
@@ -415,18 +415,15 @@ def self.cached_pd_count
   end
 
   def self.get_popular_works
-    Rails.cache.fetch('m_popular_works', expires_in: 48.hours) do
-      evs = Ahoy::Event.where(name: 'view').where("JSON_EXTRACT(properties, '$.type') = 'Manifestation'").where(
-        'time > ?', 1.month.ago
-      )
-      pop = {}
-      evs.each do |x|
-        mid = x.properties['id']
-        pop[mid] = 0 unless pop[mid].present?
-        pop[mid] += 1
-      end
-      sorted_pop_keys = pop.keys.sort_by { |k| pop[k] }.reverse
-      Manifestation.find(sorted_pop_keys[0..9])
+    Rails.cache.fetch('m_popular_works', expires_in: 24.hours) do
```

Comment:
Used same approach as for Authorites. I've also changed cache period to 24 hours instead of 48.